acceptance/indy_acceptance/indy-tests/libraries/constant.py

```diff
@@ -0,0 +1,60 @@
+
+"""
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+
+Containing all constants that are necessary to execute test scenario.
+"""
+
+
+class Colors:
+    """
+    Class to set the colors for text.
+    Syntax:  print(Colors.OKGREEN +"TEXT HERE" +Colors.ENDC)
+    """
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'  # Normal default color.
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class Roles:
+    """
+    Class to define roles.
+    """
+    TRUSTEE = "TRUSTEE"
+    STEWARD = "STEWARD"
+    TRUST_ANCHOR = "TRUST_ANCHOR"
+    TGB = "TGB"  # obsolete.
+    NONE = ""
+
+
+class Constant:
```

Comment:
In Python there is no necessity to use classes as containers of static members as it would be in Java.
In Python a module is a natural container for static data and functions.

Reply:
Updated. Please refer to the latest code.